include/gazebo_mavlink_interface.h

```diff
@@ -339,6 +343,20 @@ class GazeboMavlinkInterface : public ModelPlugin {
   common::Time last_imu_time_;
   common::Time last_actuator_time_;
 
+  ///! Enumeration to use on the bitmask in HIL_SENSOR
+  enum SensorSource
+  {
+    ACCEL	= 0x0007,
```

Comment:
I think this is compiled with C++17 and you can use binary notation which makes this clearer. E.g. `0b111000.

Reply:
right that sounds good